src/lib/containers/QueryCount.tsx

```diff
@@ -2,30 +2,23 @@ import React from 'react'
 import { SynapseConstants } from '../utils/'
 import { parseEntityIdFromSqlStatement } from '../utils/functions/sqlFunctions'
 import useGetQueryResultBundle from '../utils/hooks/SynapseAPI/entity/useGetQueryResultBundle'
-import {
-  FacetColumnValuesRequest,
-  QueryBundleRequest,
-} from '../utils/synapseTypes/'
+import { Query, QueryBundleRequest } from '../utils/synapseTypes/'
 
 export type QueryCountProps = {
-  sql: string
-  selectedFacets?: FacetColumnValuesRequest[]
+  query: Query
   parens?: boolean
 }
```

Comment:
QueryCount just shows the total results of a given query. It used to only take SQL because a query with a filter on a locked column previously applied the filter through SQL. Since the filter is now applied via a `QueryFilter`, this component needs to accept the whole `Query` object to show the correct value.